src/Service.Tests/dab-config.MsSql.json

```diff
@@ -10,7 +10,8 @@
   "runtime": {
     "rest": {
       "enabled": true,
-      "path": "/api"
+      "path": "/api",
+      "base-route": ""
```

Comment:
Is this change (and on the MySQL and PostgreSQL files) the result of running the `config-generator` scripts? We need to ensure that that is the way these files are generated so that we avoid situations where they get out of sync and can cause problems on the build server (which runs them).

Reply:
Outdated code again. But to answer your question, yes I ran the scripts locally and this was generated via that. The config file generated had a large diff because of ctrl characters i guess but this was the only change that had to be incorporated in.